src/components/ToggleButtons/ToggleButtons.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/prop-types */
+import { useState } from 'react';
+import Button from '../Button/Button';
+import { FlexContainer } from '../FlexContainer/FlexContainer';
+import './ToggleButtons.css';
+
+const ToggleButtons = ({ filterByCategory }) => {
+    const [selectedButton, setSelectedButton] = useState('dessert');
```

Comment:
In general you don't need to store selected categories here as you need to pass it into the parent component to apply filter.